projects/mysadna2/src/app/app.component.html

```diff
@@ -1,10 +1,10 @@
+<nav class="app-navigation">
+  <a [routerLink]="'/projects'">Projects</a>
+  <a [routerLink]="'/users'">Users</a>
+</nav>
 <div style="text-align:center">
```

Comment:
Please, try always put css(scss) code in scss files only. Not in html.
The only exception is dynamic binding. Example:
```
<div [style.width.px]="blockWidth"></div>
```
But it's good to not use dynamic binding too if possible.